rows.go

```diff
@@ -31,49 +31,71 @@ type Row struct {
 
 // Cell represents the data in a single cell as a consumable format.
 type Cell struct {
-	Column string // E.G   A, B, C
-	Row    int
-	Value  string
+	Column      string // E.G   A, B, C
+	ColumnIndex int
```

Comment:
The more i'm thinking about this, the more convinced I am that `ColumnIndex` should be a function on the `Cell` type, as the value is purely derived. 

i.e. 

```go
func (c *Cell) ColumnIndex() int {
   ...
}
```

Reply:
I like that. I'll update the relevant PR :)